scripts/check_codersdk_imports.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This file checks all codersdk imports to be sure it doesn't import any packages
+# that are being replaced in go.mod.
+
+set -eu
+# shellcheck source=scripts/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+cdroot
+
+set +e
```

Comment:
Why do you need to re-enable errors?

Reply:
If the grep returns no matches in `conflicts`, the whole script will fail and return no output